lib/unzipWith/index.js

```diff
@@ -12,9 +12,9 @@ export default curry2(function unzipWith(cb, args) {
   }
   var j = 0
   while (j < max) {
-    var arr = []
+    var arr = new Array(args.length)
```

Comment:
Maybe, I could be wrong, but creating new Instance is slower, when using `new` keyword. But it's exactly lighter for several bytes)